### PR TITLE
EZP-31851: ez_render Twig helpers

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-31851
| Versions      | master (3.2)

### CODE REVIEW

File: docs/guide/twig_functions_reference.md [hunk 1]
```diff
@@ -326,6 +329,73 @@ If the Content item does not have a translation in the current language, the mai
...
+It is rendered by default using the `embed` view.
+
+#### Prototype and Arguments
+
+`ez_render_content ( eZ\Publish\API\Repository\Values\Content\Content content [, string method, string viewType] ) : string`
+
+|Argument name|Type|Description|
+|------|------|------|
+|`content`|`eZ\Publish\API\Repository\Values\Content\Content`|Content item to render.|
+|`method`|`string`|[Rendering method](#rendering-methods). One of: `inline`, `subrequest`, `esi`. Default is `inline`.|
```
Comment (dabrt) [inline docs/guide/twig_functions_reference.md:379]: See my comment for line 353

File: docs/guide/twig_functions_reference.md [hunk 2]
```diff
@@ -326,6 +329,73 @@ If the Content item does not have a translation in the current language, the mai
...
+It is rendered by default using the `embed` view.
+
+#### Prototype and Arguments
+
+`ez_render_location ( eZ\Publish\API\Repository\Values\Content\Location location [, string method, string viewType] ) : string`
+
+|Argument name|Type|Description|
+|------|------|------|
+|`location`|`eZ\Publish\API\Repository\Values\Content\Location`|Location of the Content item to render.|
+|`method`|`string`|[Rendering method](#rendering-methods). One of: `inline`, `subrequest`, `esi`. Default is `inline`.|
```
Comment (dabrt) [inline docs/guide/twig_functions_reference.md:396]: Ditto

**Review by dabrt — APPROVED** [review]
Please see my comments

File: docs/guide/twig_functions_reference.md [hunk 3]
```diff
@@ -326,6 +329,73 @@ If the Content item does not have a translation in the current language, the mai
...
+|`location`|`eZ\Publish\API\Repository\Values\Content\Location`|Location of the Content item to render.|
+|`method`|`string`|Rendering method. One of: `inline`, `subrequest`, `esi`. Default is `inline`.|
+|`viewType`|`string`|[View type](templates.md#full-line-and-other-views).|
+
+#### Rendering methods
+
+The following rendering methods are available out of the box for the `ez_render()`, `ez_render_content()`,
+and `ez_render_location()` functions:
+
+- `inline` - renders the Content item without using a request
```
Comment (DominikaK) [inline docs/guide/twig_functions_reference.md:361]:
```suggestion
- `inline` - (default) renders the Content item without using a request
```

File: docs/guide/twig_functions_reference.md (hunk 1 above)
Comment (DominikaK) [inline docs/guide/twig_functions_reference.md:379]:
```suggestion
|`method`|`string`|[Rendering method](#rendering-methods). One of: `inline`, `subrequest`, `esi`.|
```

File: docs/guide/twig_functions_reference.md (hunk 2 above)
Comment (DominikaK) [inline docs/guide/twig_functions_reference.md:396]:
```suggestion
|`method`|`string`|[Rendering method](#rendering-methods). One of: `inline`, `subrequest`, `esi`.|
```

File: docs/guide/twig_functions_reference.md [hunk 4]
```diff
@@ -326,6 +329,73 @@ If the Content item does not have a translation in the current language, the mai
...
+`ez_render ( eZ\Publish\API\Repository\Values\Content\Content content|
+             eZ\Publish\API\Repository\Values\Content\Location location
+             [, string method,
+             string viewType] ) : string`
+
+|Argument name|Type|Description|
+|------|------|------|
+|`content`|`eZ\Publish\API\Repository\Values\Content\Content`|Content item to render.|
+|`location`|`eZ\Publish\API\Repository\Values\Content\Location`|Location of the Content item to render.|
+|`method`|`string`|Rendering method. One of: `inline`, `subrequest`, `esi`. Default is `inline`.|
```
Comment (DominikaK) [inline docs/guide/twig_functions_reference.md:353]:
```suggestion
|`method`|`string`|[Rendering method](#rendering-methods). One of: `inline`, `subrequest`, `esi`. Default is `inline`.|
```

[the remark this answers is not in the document]
Comment (DominikaK) [reply docs/guide/twig_functions_reference.md:353]: This is reference, do I prefer to have all the information available at a glance, but moved the 'default' information to method description as suggested.

**Review by Nattfarinn — APPROVED** [review]

**Pull request merged**